src/libasr/CMakeFiles/CMakeDirectoryInformation.cmake

```diff
@@ -0,0 +1,16 @@
+# CMAKE generated file: DO NOT EDIT!
+# Generated by "Unix Makefiles" Generator, CMake Version 3.23
```

Comment:
Should this be checked in to the repository? Looks like the whole CMakeFiles directory is checked in on accident.

Reply:
Ah! I did `git clean -fdx` before committing. Still this made into the commit. Removing.

Reply:
Fixed.